src/main.c

```diff
@@ -508,11 +541,25 @@ int main(int argc, char* argv[]) {
       goto out;
     }
     jq_set_attr(jq, jv_string("PROGRAM_ORIGIN"), jq_realpath(jv_string(dirname(program_origin))));
+    if (jv_object_has(jv_copy(program_arguments), jv_string("ARGS"))) {
+      jv_free(ARGS);
```

Comment:
I checked what happens when you repeat an arg: in `master` the first one wins, so I decided I should switch to using an object and not set when a key already existed.  As to `ARGS`, I figured "if the user really wants a different `ARGS`, why not?", but I'm actually more inclined to not let them.